pymeasure/instruments/agilent/agilent33500.py

```diff
@@ -56,58 +365,58 @@ class Agilent33500(Instrument):
 
         generator = Agilent33500("GPIB::1")
 
-        generator.shape = 'SIN'                 # Sets the output signal shape to sine
-        generator.frequency = 1e3               # Sets the output frequency to 1 kHz
-        generator.amplitude = 5                 # Sets the output amplitude to 5 Vpp
-        generator.output = 'on'                 # Enables the output
+        generator.ch[n].shape = 'SIN'                 # Sets the output signal shape to sine
+        generator.ch[n].frequency = 1e3               # Sets the output frequency to 1 kHz
+        generator.ch[n].amplitude = 5                 # Sets the output amplitude to 5 Vpp
+        generator.ch[n].output = 'on'                 # Enables the output
```

Comment:
I would replace `n` with an actual channel number to make the code working as an example.
Same comment in other example code below.

Reply:
Modified in https://github.com/pymeasure/pymeasure/pull/763/commits/3e71c1eee00e7ff48596b6418855cf6a67fe831e 
I added an extra comment about the default channel